contracts/core/EntryPoint.sol

```diff
@@ -219,9 +222,9 @@ contract EntryPoint is IEntryPoint, StakeManager {
 
         IPaymaster.PostOpMode mode = IPaymaster.PostOpMode.opSucceeded;
         if (callData.length > 0) {
-
-            (bool success,bytes memory result) = address(mUserOp.sender).call{gas : callGasLimit}(callData);
+            bool success = Exec.call(mUserOp.sender, 0, callData, callGasLimit);
```

Comment:
please change the param from 'calldata callData` to `memory callData`...
it saves 100 gas (currently, this feature cost 230 gas. with the above fix its only 130)

Reply:
should one make use of memory instead of calldata everywhere for external functions?

Isn't this true? : To save a significant amount it is better to use calldata in the arguments of an external function

Reply:
YMMV... Solidity optimization is funny. It has to copy into memory in order to be used. "memory" in parameter requires early copy for into memory, but in some cases Solidity gets confused and copy it again.. so it is better to try the options: make a change, and run "yarn gascalc" to see the effect on some basic scenarios. 
In this specific case, making the "context" parameter memory instead of calldata actually takes more gas... go figure.